ca_gazebo/launch/create_two_floors_house.launch

```diff
@@ -1,7 +1,7 @@
 <launch>
   <arg name="virtual_wall_x" default="0.2"/>
   <arg name="virtual_wall_y" default="-1.2"/>
-  <arg name="virtual_wall_yaw" default="1.57"/>
+  <arg name="virtual_wall_yaw" default="-1.57"/>
```

Comment:
Was this change necessary at the end? I don't think so. Please double check.

Reply:
In the context of what the virtual_wall is supposed to do it's necessary to block the correct opening of the room. Not doing this makes having a virtual wall blocking a real wall and not an opening.